LICENSE

```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Is it a mistake that licence file is deleted?

Reply:
No we will not produce an entrypoint for this repository since multiple licence file are embeded in the container by NVIDIA